### DISCUSSION TITLE
GC Internal (Laser Bear) Hotkey Question

### DISCUSSION THREAD

**Discussion by Ghost-Mop** [darthcloud/BlueRetro#498] [Q&A]
Hi, 

Just installed my internal gc blueretro board and loving it after I resolved the digital+analog combo preset option on the configurator. (If you aren't aware, this preset lets you air-dodge, wavedash, etc on an Xbox One/S controller and probably others)
I've been goofing around with that but noticed there doesn't appear to be an obvious way to edit the Button Combination Functions aka "hot key combos."

The following hotkey, written in PSX terms (from /darthcloud/BlueRetro/wiki#5---button-combinations-functions ), is the one I would want to change on a global or GAMEID basis as it causes a problem with Super Smash Bros: Melee
Disconnect controller (+ power off internal mod):
Main Left Trigger (L2) + Main Right Trigger (R2) + Middle Right (Start) + Face Down (X)

Because, on Melee (in Gamecube terms), the quickest way to exit a match is to press 
Main Left Trigger (L) + Main Right Trigger (R) + Middle Right (Start) + Face Down (A)

This overlap isn't ideal SO if there exists a way to change this particular key myself I would love to be pointed in that direction. If its just not something that has been implemented yet that's totally cool too just curious.

Quizically Yours,
Ghost Mop

**Comment by darthcloud** [comment]
Hotkey are not remappable.

If you use a controller with analog trigger, I think that if you only slightly press the trigger it will triggers the blueretro hotkey.

If you fully press the trigger it should do the melee quit. Just make sure to release A first so that blueretro hotkey doesn't get trigger by releasing the triggers first.

I think this is what I  observed the other day when playing melee but I could remember wrong.

**Reply by Ghost-Mop** [reply]
Just tested this, releasing either trigger, both triggers, “start,” or “A” first results in an immediate sleep from the console. At time of initiating the hotkey, all triggers were fully depressed before adding A and Start.

Would this be a result of using the combine analog and digital after 95% preset?

**Reply by mynameistc** [reply]
> Hotkey are not remappable.
> 
> If you use a controller with analog trigger, I think that if you only slightly press the trigger it will triggers the blueretro hotkey.
> 
> If you fully press the trigger it should do the melee quit. Just make sure to release A first so that blueretro hotkey doesn't get trigger by releasing the triggers first.
> 
> I think this is what I observed the other day when playing melee but I could remember wrong.

I'm using two different Xbox Series controllers (one elite one regular) and this does not work for me.  

It's frustrating that a button combination that I'll hopefully never have to use is getting in the way of the only way to quit a match in Melee.  
Is there any way you could consider changing this mapping?  Possibly adding an additional button to the macro?  I'll throw a bounty on this (starting at $50) just to get this changed.
Edit: Or have an option to remove macros altogether even?
